mycpp/gc_dict.h

```diff
@@ -155,15 +147,13 @@ class Dict {
   int capacity_;  // number of entries before resizing
 
   // These 3 slabs are resized at the same time.
-  Slab<int>* entry_;  // NOW: kEmptyEntry, kDeletedEntry, or 0.
-                      // LATER: indices which are themselves indexed by // hash
-                      // value % capacity_
+  Slab<int>* index_;  // kEmptyEntry, or a valid index into keys_/values_
```

Comment:
this can still be kDeletedEntry

Reply:
Ah nice catch. Leftover from an old experiment